modules/sms_email_reminder/cron_email_notification.php

```diff
@@ -34,7 +33,7 @@
     $_GET['site'] = isset($args[1]) ? $args[1] : 'default';
 }
 if (php_sapi_name() === 'cli') {
-//    $_SERVER[‘HTTP_HOST’] = ‘localhost’;
+    // $_SERVER[‘HTTP_HOST’] = ‘localhost’;
     $_SERVER['HTTP_HOST'] = 'localhost';
```

Comment:
yep, you need this. Was the whole purpose you got into this huh, Luis!
Let me know when ready and I'll merge for you.
I'm assuming you've well tested where I'm not currently setup to do so.

Reply:
Yes Jerry, it was because of this (the cron). I have already tested it on Linux (Rocky Linux, Openemr 6.1, php 8.1) and Windows 11 (xampp, identical) and it works fine. I was correcting and cleaning the code today. Thank you.